client/app/hosting/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -3,427 +3,340 @@
+                                    <span data-ng-bind=":: convertBytesSize(hosting.quotaUsed.value, hosting.quotaUsed.unit, 2) + ' / ' + convertBytesSize(hosting.quotaSize.value, hosting.quotaSize.unit)"></span>
                                 </li>
                             </ul>
-                        </div>
-                    </div>
-
-                </div>
+                        </oui-tile-description>
+                    </oui-tile-definition>
+                    <oui-tile-definition
+                        data-term="{{:: 'hosting_dashboard_service_boost' | translate }}"
+                        data-ng-if="!hosting.isCloudWeb">
+                        <oui-tile-description>
+                            <div data-ng-if="hostingProxy.boostOffer">
+                                <span class="fa fa-check text-success" aria-hidden="true"></span>
+                                <span data-ng-bind="'(' + hostingProxy.boostOffer + ')'"></span>
+                            </div>
+                            <div data-ng-if="!hostingProxy.boostOffer">
+                                <span data-translate="hosting_dashboard_service_boostOffer_disabled"
+                                      data-ng-if="hosting.availableBoostOffer && hosting.availableBoostOffer.length > 0"></span>
+                                <span data-translate="hosting_dashboard_service_boostOffer_unavailable"
+                                      data-ng-if="!hosting.availableBoostOffer || hosting.availableBoostOffer.length === 0"></span>
+                            </div>
+                        </oui-tile-description>
+                    </oui-tile-definition>
+                    <oui-tile-definition
+                        data-term="{{:: 'hosting_dashboard_service_datacenter' | translate }}">
+                        <oui-tile-description>
+                            <span
+                                data-ng-bind="(('hosting_dashboard_service_datacenter_'+ hostingProxy.datacenter | translate) == 'hosting_dashboard_service_datacenter_'+ hostingProxy.datacenter)? hostingProxy.datacenter: ('hosting_dashboard_service_datacenter_'+ hostingProxy.datacenter | translate)"></span>
+                        </oui-tile-description>
+                    </oui-tile-definition>
+                    
+                    <oui-tile-definition
+                        data-ng-if="!hosting.isCloudWeb"
+                        data-term="{{:: 'hosting_dashboard_service_main_domain' | translate }}">
+                        <oui-tile-description>
+                            <span class="word-break" data-ng-bind="hosting.serviceName"></span>
+                        </oui-tile-description>
+                        <oui-action-menu data-compact data-align="end"
+                                         data-ng-if="['KIMSUFI_2015','__60_FREE','DEMO_1_G','START_1_M','START_10_M','_ASPFREE'].indexOf(hosting.offer) === -1">
```

Comment:
It would be better to handle this test in a controller method, (and use `.includes()` by the way)